media-video/ffdiaporama/ffdiaporama-2.2_pre20140701-r3.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-COMMIT="3c25c893a94dd3f88a7515437c42cd5628220625"
+COMMIT="0557dc2d2489b88c813c695b0d75107be82292f8"
```

Comment:
Are you sure this doesn't result in tarball manifest change?

1. Please do a new release (with new snapshot date, and please recheck the version number).
2. Please remanifest (after deleting old distfile from distfiles directory). I can recommend to use `pkgdev manifest -f` from the `pkgdev` package.

Reply:
Sorry, I forgot to remove the previous tarball from distfiles. It is ok now.

I am not sure it is necessary to change the name of the ebuild. This software has been dead upstream for a long time now and writing new patches was getting harder and harder so I decided to move the source code to my own github but the final result is the same, it is a patched version of the last upstream development version (2.2_devel 20140701).

This pull request is just meant to correct the one merged yesterday, nothing more.

Reply:
```!!! Fetched file: ffdiaporama-2.2_pre20140701.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      63271629
!!! Expected: 63270245
Refetching... File renamed to '/var/cache/distfiles/ffdiaporama-2.2_pre20140701.tar.gz._checksum_failure_._jb2p8o6'
```

It's fine not to revbump but you need to rename the distfile, I think.

Reply:
OK, I did not have all the consequences in mind, the name has indeed to be different. So I have just modified the date

Reply:
no problem, thanks!